luna/class1-1/#4 1008.py

```diff
@@ -0,0 +1,6 @@
+A, B = input().split()
+
+A = int(A)
+B = int(B)
+
+print(A/B)
```

Comment:
```suggestion
a, b = map(int, input().split())
print(a / b)
```

변수 이름을 대문자로 사용하지 않는것을 추천드립니다
- 보편적으로 대문자는 상수를 의미합니다. (예: API_KEY)

map(int, input().split())을 사용하여 형변환을 한번에 할 수 있습니다. 
- 해당 방식을 통해 보다 파이썬스럽게 코드를 사용할 수 있습니다.

Reply:
문제에서  '첫 줄에 A와 B가 주어진다'라고 되어있을 때에도
답안 코드를 작성할 때 A와 B 대신 a b 등으로 사용해야 하나요?

Reply:
변수 이름을 소문자로 사용하는것은 필수적으로 따라야하는 규칙이 아니라
원하시는대로 해도상관 없습니다.
다만 보편적인 방식을 사용하는게 추후 협업시에도 서로가 작성한 코드를 읽기 쉬워서 추천 드린 내용입니다.
보다 자세한 보편적 규칙을 알고싶으시다면 [구글 파이썬 스타일 가이드](https://yosseulsin-job.github.io/Google-Python-Style-Guide-kor/#s3.1) 읽어보시는거 추천드리고
(조금 양이 많아 시간 나실때 천천히 읽어보시길 추천드립니다.)
변수 이름과 관련된 스타일 규칙은 [여기](https://yosseulsin-job.github.io/Google-Python-Style-Guide-kor/#s3.16) 있습니다.

Reply:
고맙습니다~